[3029] copy update search suggestion when searching for a salaried course when there are 3 salaried courses

gpeng opened the pull request:
### Context

If a user search for salaried courses returns 3 or less results we show them suggested search links. The first link should include salaried and non-salaried courses at a radius that returns more courses than the original search. If widening the radius further returns more results we should show a link to that too. The subsequent links should respect the original filter and only include salaried courses.

If an increment of the radius only returns the same number of courses as the previous radius we don't show the link.

### Changes proposed in this pull request

Update the text of the links to better describe the searches that they link to.

### Guidance to review

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Product review

defong commented on app/view_objects/suggested_search_link.rb:
```diff
@@ -34,4 +35,11 @@ def suggested_search_link_parameters(radius:)
   end
+
+  def text_course_text
+    return "non-salaried course" if including_non_salaried
```
Not too sure about the wording...

non-salaried

TobyRet replied:
If Paul Genders is happy, then I'm happy

defong replied:
pretty sure that terminology was leaked 

https://github.com/DFE-Digital/find-teacher-training/pull/226#discussion_r398030780

defong replied:
![image](https://user-images.githubusercontent.com/470137/78788237-6b986100-79a3-11ea-84ed-22f7206ae5f8.png)

Until this is bottom out it, it is not going anywhere far...


`non-salaried courses` does not mean `Courses with and without salary`
The suggested search number does not tally up/follow through 

LHS url
https://s121d02-3029-find-as.azurewebsites.net/results?fulltime=false&funding=8&hasvacancies=true&l=1&lat=55.953252&lng=-3.188267&loc=Edinburgh%2C+UK&lq=Edinburgh%2C+UK&parttime=false&qualifications[]=QtsOnly&qualifications[]=PgdePgceWithQts&qualifications[]=Other&rad=5&senCourses=false&sortby=2

gpeng replied:
I think this is probably a bug in one of the queries rather than an issue with copy or semantics there of. I'll do another card to look at this so as we can just get the copy sorted in this PR.

TobyRet commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Copy update: Search suggestion when searching for a salaried course when there are <3 salaried courses](https://trello.com/c/RJNa3CMp/3029-copy-update-search-suggestion-when-searching-for-a-salaried-course-when-there-are-3-salaried-courses)

gpeng commented:
![image](https://user-images.githubusercontent.com/5216/79234971-ec070800-7e62-11ea-8cfb-f4386c5d9cfd.png)


@defong We've updated some functionality and split a bit of the text out of the link. Could you have another look before we merge? I'll create a new card for the count mismatch bug.
